src/coreclr/pal/src/thread/process.cpp

```diff
@@ -3359,6 +3360,8 @@ PROCAbort()
 
     PROCCreateCrashDumpIfEnabled();
 
+    SEHCleanupSignals();
```

Comment:
Why do we need to call SEHCleanupSignals here? Our policy is to not cleanup on process exit, unless it is absolutely necessary.

Reply:
The SIGABRT handler that I added needs to be removed in order to prevent recursion when abort() is called.

Reply:
Is the recursion going to be a problem for other places (e.g. in 3rd party libraries) that may call `abort`?

Reply:
It shouldn't because for those abort()s we will catch them, generate the coredump, uninstall the signal handler, and then return from the signal handler in invoke_previous_action in signal.cpp. The normal path through that function doesn't call PROCAbort (fails through to the PROCCreateCrashDumpIfEnabled at the end).  

Even if PROCAbort is called in the invoke_previous_action paths that do, I assume calling abort() again after uninstalling the signal handlers will abort with any problems.

Reply:
I've tested this by pinvoking to abort() in my test app.

Reply:
Ok, I would limit the cleanup to just SIGABRT.